packages/react-union-scripts/scripts/startDevServer.js

```diff
@@ -49,7 +49,7 @@ function startDevServer() {
 		'You can start DEV Sever only for one module at the same time.'
 	);
 
-	const webpackConfig = configs[0];
+	const webpackConfig = configs[0][0];
```

Comment:
Maybe it should not be an array, but instead an object with client, server properties.